root/etc/e-smith/templates/getmailrc/10retriever

```diff
@@ -4,8 +4,21 @@
     my $server = $account->prop('Server') || 'localhost';
     my $user = $account->prop('Username') || '';
     my $pass = $account->prop('Password') || '';
+    my $mailbox = $account->prop('Mailbox') || '';
+
     $OUT .= "type = $retr\n";
     $OUT .= "server = $server\n";
     $OUT .= "username = $user\n";
     $OUT .= "password = $pass\n";
+
+    if (($retr eq 'SimplePOP3Retriever') || ($retr eq 'SimplePOP3SSLRetriever')) {
+        # Can't specify mailboxes for POP3-retrievers.
+    } elsif ((uc $mailbox) eq 'ALL' ) {
+        $OUT .=  "mailboxes = ALL\n";
+    } elsif ($mailbox ne '') {
+        $mailbox =~ s/,/\",\"/g;
```

Comment:
Hi Davide, thank you for your feedback. I'll look into it, might take me some time to figure out nethserver works internaly. Please be patient ;-)